e2e/atlas/helper_test.go

```diff
@@ -128,6 +129,40 @@ func deployCluster() (string, error) {
 	return clusterName, nil
 }
 
+func newAvailableRegion(projectID, tier, provider string) (string, error) {
+	cliPath, err := e2e.Bin()
+	if err != nil {
+		return "", err
+	}
+	cmd := exec.Command(cliPath,
+		atlasEntity,
+		clustersEntity,
+		"availableRegions",
+		"ls",
+		"--provider", provider,
+		"--tier", tier,
+		"--projectId", projectID,
+		"-o=json")
+	cmd.Env = os.Environ()
+	resp, err := cmd.CombinedOutput()
+
+	if err != nil {
+		return "", err
+	}
+
+	var cloudProviders mongodbatlas.CloudProviders
+	err = json.Unmarshal(resp, &cloudProviders)
+	if err != nil {
+		return "", err
+	}
+
+	if len(cloudProviders.Results) == 0 || len(cloudProviders.Results[0].InstanceSizes) == 0 {
+		return "", errors.New("no regions available")
+	}
+
+	return cloudProviders.Results[0].InstanceSizes[0].AvailableRegions[0].Name, nil
```

Comment:
could we `return InstanceSizes[0].Name, InstanceSizes[0].AvailableRegions[0].Name, nil`, like return both and avoid to hard code the instance size in the test?

Reply:
mh.. I don't understand why. The purpose of this function is to return the first region available, I do not see why it should also return the `InstanceSize` which is given as input with `tier`

Reply:
trying to see if we can get default tier, so is the tier flag mandatory? this is not a blocker just like previous comments in this pr trying to reduce the amount of values that are hardcoded

Reply:
no, the tier is not mandatory. If we don't provide it, we get the regions for all instance sizes. Are you suggesting not passing the tier to this function and return `return InstanceSizes[0].Name, InstanceSizes[0].AvailableRegions[0].Name, nil`? (If yes) I believe that this strategy may work since all instance sizes have the same regions, but this may change in the future.

I am going to merge this PR and open a new one if there are follow-ups here.

Reply:
> Are you suggesting not passing the tier to this function and return return InstanceSizes[0].Name, InstanceSizes[0].AvailableRegions[0].Name, nil?

Yes that's what I say, it be better in the long wrong to just have a this function instead of hardcoded values

Reply:
I will open a PR to address this comment.